ts/editable/ContentEditable.svelte

```diff
@@ -70,17 +61,6 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         &:focus {
             outline: none;
         }
-        &.empty::before {
-            content: var(--description);
-            opacity: 0.4;
-            cursor: text;
-            /* stay on single line */
-            position: absolute;
-            max-width: 95%;
-            overflow-x: hidden;
-            white-space: nowrap;
-            text-overflow: ellipsis;
```

Comment:
The reason I restricted the description width was to prevent the pseudo-element from expanding the editable. Your implementation doesn't expand it either, but you should still cut off the text-overflow:

![image](https://user-images.githubusercontent.com/62722460/174972504-4bc78074-94fa-4736-90c4-ef39a1b496ca.png)